readme.md

```diff
@@ -89,6 +89,7 @@ $ cpy --help
 
   Example
     $ cpy 'src/*.png' dist
+    $ cpy 'src/*.png' '!src/*.ico' dist
```

Comment:
Be that as it may, I think the example should be the same as the one in the usage section.
`'src/*.png' '!src/*.ico'` is in my opinion not very useful.